### DISCUSSION TITLE
Copy timeline output as HTML

### DISCUSSION THREAD

**Discussion by coolhome** [Kong/insomnia#3700] [Ideas]
I think having the ability to copy the curl timeline output as HTML would be beneficial when using chat clients like Microsoft Teams. This feature would bring the color highlighting with it. If implemented it should probably be a preference.

**Comment by gschier** [comment]
I think it's fine to enable rich copy by default. You can always use Cmd+Shift+V (or Windows/Linux alternative) to paste unformatted if needed.

**Comment by gschier** [comment]
I'm not actually sure how to implement this though but it seems like a good issue for a first-time contributor to tackle. Perhaps Codemirror (the text editor component we use) has an easy way to enable this.
